src/wfs/src/main/java/org/geoserver/wfs/json/ComplexGeoJsonWriter.java

```diff
@@ -618,6 +648,12 @@ private void encodeSimpleAttribute(
         jsonWriter.endObject();
     }
 
+    private void key(String name) {
```

Comment:
Please add documentation, this pattern is already strange enough :yum:.

Reply:
added